### ISSUE TITLE
The metrics on the video are different from the metrics on the filmstrip and the summary page

### ISSUE THREAD

**Issue by UladzislauCharkouski** [sitespeedio/sitespeed.io#2957]
We faced with a problem when the same metrics on the video and on the filmstrip are different.
![2020-04-14_12h17_42](https://user-images.githubusercontent.com/61200221/79213216-76467080-7e51-11ea-9458-61b4b27061a1.png)
![2020-04-14_12h18_35](https://user-images.githubusercontent.com/61200221/79213221-78103400-7e51-11ea-8274-0647cebbdec4.png)
![2020-04-14_12h25_44](https://user-images.githubusercontent.com/61200221/79213225-79d9f780-7e51-11ea-8082-c4d8463c8914.png)

It looks like the most accurate metric in the video.
About 200 measurements were carried out, and in about 15-20% of them there are these differences and sometimes they reach more than 100 ms.
The docker run command looks like this: - docker run --network=cable --rm -v "$PWD":/sitespeed.io sitespeedio/sitespeed.io:12.5.0 -n50 --multi $SCRIPT --budget.configPath budget.json --budget.output json --resultBaseURL $RESULT_BASE_URL --logToFile true
Testing is performed in linux environment using a js script:
module.exports = async function(context, commands) {
    try {
        await commands.navigate('the URL isn’t public','LoginPage');
	await commands.addText.byId('email','home__formMemberEmail_username_username');
	await commands.addText.byId('pass','home__formMemberEmail_pass1_password');
        await commands.click.bySelectorAndWait("button[type=submit]");
	await commands.wait.byXpath('//my-profile-bar/amw-menu-drop-down/button', 60000);
	await commands.click.byXpathAndWait('//my-profile-bar/amw-menu-drop-down/button');
	await commands.click.bySelectorAndWait('#cdk-overlay-0 > div > div > div > div > div > div:nth-child(4) > a');
	await commands.wait.byXpath('//amw-card/div', 60000);
	await commands.measure.start('business-information');
	await commands.click.bySelectorAndWait('div[class=amw-card]');
	return commands.measure.stop();
    } catch (e) {
    }
};

Our application works on the basis of the angular as single page application.
Could you tell me how to solve this problem and align the metrics?

**Comment by soulgalore** [comment]
Hi @UladzislauCharkouski thanks for creating the issue. Its hard to see in the blurred images but my guess is that the difference are on the summary level (not clicking on a specific run?)? 

The metrics are median on the summary level and the filmstrip are a specific run (I think the run that had the median SpeedIndex) so they will probably not be the same there.

But then also the video and the filmstrip show different metrics (on the same level right), that seems to be that we pick the wrong run, we should make sure we show the same run.

FYI: try to make an uneven number of runs so instead 50 do 49 or 51, then the median will be a specific run, else the median is the average of two runs.

**Comment by UladzislauCharkouski** [comment]
Hi @soulgalore, thanks for the answer, first of all I want to say that screenshots from a specific run. 
I took a lot more measurements and I seem to have found the reason for this behavior. This happens if one of the measurements fails. 
For example, I made a measurement in 50 runs and one of the measurements (the 14th run in my case) failed with an error: "ERROR: Couldn't execute script named CUSTOM error: JavascriptError: javascript error: Cannot read property 'click' of null"
![2020-04-21_11h01_45](https://user-images.githubusercontent.com/61200221/79848165-625abb80-83c9-11ea-9df4-1b6eead831cb.png)

Then at the end, error appears when saving results: "ERROR: Could not read filmstrip dir Error: ENOENT: no such file or directory, scandir '/sitespeed.io/sitespeed-result/bc_business-information_js/2020-04-17-15-49-25/pages/myapp.com/my-account/business-information/data/filmstrip/14''
![2020-04-21_11h05_58](https://user-images.githubusercontent.com/61200221/79842355-5f5bcd00-83c1-11ea-8371-8326c2dcb583.png)

As a result, from the 15th run, the video and filmstrip no longer match in all of the following runs (match only if the time of the metrics was the same in two consecutive measurements). 
For example: the filmstrip on run 15 and its video on run 16 and etc.
If all measurements are successful, then all the videos correspond to the filmstrip.
It seems that after an unsuccessful measurement, the data in the folder with results are recorded incorrectly, but why I don’t know. Maybe you have any suggestions?

p.s. sorry for blur, but I can't distribute any project information
